views/templates/hook/admin_order.tpl

```diff
@@ -148,10 +163,12 @@
                         <label class="control-label col-lg-3">{l s='Hour range from' mod='clickline'}: <span style="color:red;">*</span></label>
                         <div class="col-lg-2">
                             <select name="hour_range_from">
-                                <option value="1" {if $params['hour_range_from'] eq '1'}selected="selected"{/if}>09:00 - 11:00</option>
+                                <!--<option value="1" {if $params['hour_range_from'] eq '1'}selected="selected"{/if}>09:00 - 11:00</option>
```

Comment:
Hi @clicklin, instead of commenting this section, would it be possible to just remove it?
Thank you.